backend/seeder/main.go

```diff
@@ -141,20 +151,19 @@ func seedTestData(db *gorm.DB) {
 				if err := db.Create(&outcome).Error; err != nil {
 					log.Printf("Failed to create outcome: %v", err)
 				}
+			} else {
+				newMilestone := models.Milestone{
```

Comment:
If a kolibri instance is created, an OpenContentProvider will be automatically created + enabled. The thought being that if it exists, it will show up in the UI as a resource we can link to